pipeline/misc/summary_stats_merger.ipynb

```diff
@@ -126,12 +126,7 @@
     "- TARGET\n",
     "   - the reference summary statistic file, which the other files compare with.\n",
     "- OUTPUT\n",
-    "   - the path of an output directory for new summary statistic files\n",
-    "   - for each input sumstat file, a qced version will be generated.\n",
-    "   - The generated sumstat files will have header as \"CHR  ,   POS  ,   A0   ,   A1    ,  SNP   ,  STAT ,   SE    ,  P\" regardless of input header\n",
-    "   - The generated sumstat files will be in gz format.\n",
-    "- KEEP_AMBIGUOUS\n",
-    "   - if Ture, keep ambiguous alleles which can not be decided from flip or reverse, such as A/T or C/G. Otherwise, remove them."
+    "   - the path of an output directory for new summary statistic files"
```

Comment:
@changebio please adopt some of @hsun3163 's additional documentation text -- trivial, but I do want to establish that PR should improve, and not revert, useful changes that have been made.